pkgs/top-level/all-packages.nix

```diff
@@ -36395,7 +36395,7 @@ with pkgs;
     inherit (gnome2) libgnomecanvas;
   };
 
-  xournalpp = callPackage ../applications/graphics/xournalpp {
+  xournalpp = darwin.apple_sdk_11_0.callPackage ../applications/graphics/xournalpp {
```

Comment:
I've not seen this before, and I'm worried about introducing it. (It does evaluate and build on NixOS, but still.)

Reply:
Doesn't the darwin build work without it?

Reply:
> Doesn't the darwin build work without it?

No, it's broken without it.

Reply:
> I've not seen this before, and I'm worried about introducing it. (It does evaluate and build on NixOS, but still.)

It's kind of common way to fix packages that requires sdk 11:
```sh
grep -c "= darwin.apple_sdk_11_0.callPackage" pkgs/top-level/all-packages.nix
138
```

Reply:
Oh, I didn't know. So it's because the new version requires binutils, and binutils must come from the apple sdk? (If it's not related to the application update, it should be a separate commit: `xournalpp: add darwin support` or something.)

Reply:
No, it's not caused by binutils. Darwin build is broken on master as well:
```
xournalpp> CMake Error at cmake/find/FindCXX17.cmake:39 (message):
xournalpp>   optional header not found
xournalpp> Call Stack (most recent call first):
xournalpp>   CMakeLists.txt:80 (find_package)
```